core/Piranha.Manager/assets/src/scss/inc/_variables.scss

```diff
@@ -93,3 +92,7 @@ $nav-tabs-inverted-link-active-border-color: $nav-tabs-inverted-link-active-bg $
 // Progress bars
 
 $progress-bg: $gray-400;
+
+// Login
+
+$login-invalid-feedback-color: #f94c3b;
```

Comment:
See above comment. This change should be removed

Reply:
Just did :)